integration-tests/src/test/resources/features/webhookaction.feature

```diff
@@ -0,0 +1,51 @@
+Feature: Webhook Action tests
+
+  @webhookaction
+  Scenario: Webhook is correctly called
+    Given authenticate against Manager
+    And create a new Bridge "mybridge"
+    And the Bridge "mybridge" is existing with status "ready" within 2 minutes
+    And the Ingress of Bridge "mybridge" is available within 2 minutes
+    And Webhook site request history is cleared
+
+    And add a Processor to the Bridge "mybridge" with body:
+    """
+    {
+      "name": "myProcessor",
+      "action": {
+        "type": "Webhook",
+        "parameters": {
+            "endpoint": "https://webhook.site/${system.webhook.site.uuid}"
+        },
+        "transformationTemplate" : "{\"text\": \"hello {data.name} by {id}\"}"
+      }
+    }
+    """
+    And the list of Processor instances of the Bridge "mybridge" is containing the Processor "myProcessor"
+    And the Processor "myProcessor" of the Bridge "mybridge" is existing with status "ready" within 3 minutes
+    And the Processor "myProcessor" of the Bridge "mybridge" has action of type "Webhook" and parameters:
+      | endpoint | https://webhook.site/${system.webhook.site.uuid} |
+
+    When send a cloud event to the Ingress of the Bridge "mybridge" with path "events":
+    """
+    {
+      "specversion": "1.0",
+      "type": "webhook.site.invoked",
+      "source": "WebhookActionTestService",
+      "id": "webhook-test",
+      "data": {
+          "name": "world"
+        }
+    }
+    """
+    Then the Ingress of the Bridge "mybridge" metric 'http_server_requests_seconds_count{method="POST",outcome="SUCCESS",status="200",uri="/events",}' count is at least 1
+    Then Webhook site contains request with text "hello world by ${bridge.mybridge.cloud-event.webhook-test.id}" within 1 minute
+
+
+    When delete the Processor "myProcessor" of the Bridge "mybridge"
+    
+    Then the Processor "myProcessor" of the Bridge "mybridge" is not existing within 2 minutes
```

Comment:
Not needed as this is handled by cleanup script

Reply:
sure, I will do @sutaakar